Src/Support/Google.Apis.Auth.AspNetCore/Google.Apis.Auth.AspNetCore.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
I'd expect this to be netstandard2.0 - any reason we need netcoreapp2.0 instead?

Reply:
Yes, I'd also expect that to be netstandard2.0, not quite sure how I created the project like this.